package.json

```diff
@@ -137,12 +137,16 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "./node_modules/.bin/chokidar 'server/**/*.(js|jsx)' 'client/**/*.(js|jsx)' 'test/**/*.(js|jsx)' -c 'pkill node -f runner.js; npm run test-client ${TEST_PATH}'",
```

Comment:
You don't need `./node_modules/.bin` . npm automatically includes this in the path for npm scripts ([reference](https://docs.npmjs.com/misc/scripts#path)).